### PR TITLE
Prezento nginx

### PR DESCRIPTION
**Pull request by rafamanzo**
Enables prezento's proxing through NGINX for CentOS and Debian.

### CODE REVIEW

**Comment by do-you-dare** [comment]
Nice job :) :+1:

**Comment by danielkza** [comment]
Tested and working in Debian Jessie container (using systemd-nspawn). Very nice job :+1:

File: Rakefile [hunk 1]
```diff
@@ -48,6 +50,15 @@ namespace :debian do
     sh "docker run -t -i --volume=#{Dir.pwd}/pkgs:/root/mezuro/pkgs mezuro-debian-build bash /root/mezuro/scripts/prezento/run.sh /root/mezuro/#{kalibro_configurations_deb} /root/mezuro/#{kalibro_processor_deb}"
   end
 
+  desc 'Build the Prezento NGINX proxy configuration package for CentOS'
```
Comment (diegoamc) [inline Rakefile:53]: `CentOS` -> `Debian`

File: scripts/prezento-nginx/prezento.conf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  server_name           localhost;
+
+  access_log             /var/log/nginx/mezuro.access.log;
+  error_log              /var/log/nginx/mezuro.error.log;
+  underscores_in_headers on;
+  client_max_body_size   150m;
+
+
+  #root  /usr/share/mezuro/prezento/public/;
```
Comment (diegoamc) [inline scripts/prezento-nginx/prezento.conf:14]: Maybe erase this comment?

Comment (rafamanzo) [inline scripts/prezento-nginx/prezento.conf:14]: Yes

**Pull request merged**